leakcanary-android-core/src/main/java/leakcanary/internal/activity/screen/AboutScreen.kt

```diff
@@ -20,11 +27,30 @@ internal class AboutScreen : Screen() {
       val application = activity.application
       val appName = application.packageManager.getApplicationLabel(application.applicationInfo)
       val appPackageName = context.packageName
+
       aboutTextView.text = Html.fromHtml(
           String.format(
-              resources.getString(R.string.leak_canary_about_message), appName, appPackageName
+              resources.getString(R.string.leak_canary_about_message), appName, appPackageName,
+              getHeapDumpStatusMessage(resources)
           )
       )
     }
 
+  private fun getHeapDumpStatusMessage(resources: Resources): String {
+    return when (HeapDumpPolicy.getHeapDumpStatus()) {
```

Comment:
Nit: replace with expression body + no need to pass `resources`, since it can be accessed from the class instance.
```kotlin
private fun getHeapDumpStatusMessage() =
  when (HeapDumpPolicy.getHeapDumpStatus()) {
  ...
```

Reply:
Done for the expression body. I think I still need to pass resources, since it's resolved in the apply block after the ViewGroup is inflated. I can't access _resources_  directly from the private function.